src/scss/social-feeds/_scrollbox.scss

```diff
@@ -1,3 +1,22 @@
+$color-facebook: #3a5795;
+$color-youtube: #cd201f;
+
+.crt-social-icon {
```

Comment:
Down the road, if we develop other unique styles for our Curator feeds (besides the 'scrollbox' layouts), it might make sense to move the `.crt-social-icon` styles into a new _base.scss file (since these icon styles would be shared across all layouts)